README.md

```diff
@@ -177,18 +177,19 @@ The following table lists the configurable parameters of the Harbor chart and th
-| `core.podAnnotations`                                                       | Annotations to add to the core pod                                                                                                                                                                                                                                                                                                              | `{}`                            |
-| `core.secret`                                                               | Secret is used when core server communicates with other components. If a secret key is not specified, Helm will generate one. Must be a string of 16 chars.                                                                                                                                                                                     |                                 |
-| `core.secretName`                                                               | Fill the name of a kubernetes secret if you want to use your own TLS certificate and private key for token encryption/decryption. The secret must contain keys named: `tls.crt` - the certificate and `tls.key` - the private key. The default key pair will be used if it isn't set |                                 |
+| **Core** |
+| `core.image.repository` | Repository for Harbor core image | `goharbor/harbor-core` |
+| `core.image.tag` | Tag for Harbor core image | `dev` |
+| `core.replicas` | The replica count  | `1` |
+| `core.livenessProbe.initialDelaySeconds` | The initial delay in seconds for the liveness probe | `300` |
+| `core.resources` | The [resources] to allocate for container | undefined |
+| `core.nodeSelector`  | Node labels for pod assignment | `{}` |
+| `core.tolerations` | Tolerations for pod assignment | `[]` |
+| `core.affinity` | Node/Pod affinities | `{}` |
+| `core.podAnnotations` | Annotations to add to the core pod | `{}` |
+| `core.secret` | Secret is used when core server communicates with other components. If a secret key is not specified, Helm will generate one. Must be a string of 16 chars. | |
+| `core.secretName` | Fill the name of a kubernetes secret if you want to use your own TLS certificate and private key for token encryption/decryption. The secret must contain keys named: `tls.crt` - the certificate and `tls.key` - the private key. The default key pair will be used if it isn't set | |
+| `core.xsrfKey` | The XSRF key, must be a string of 40 chars. Will be generated automatically if it isn't specified | |
```

Comment:
Does it allow special chars like `#` `$`?  I recall it does not?

Reply:
Not find any clarification for that